src/test_ingest.py

```diff
@@ -45,40 +45,40 @@ def test_ingest_ok(self, mock_writeToElasticsearch):
         mock_formatter.get_data_objects = MagicMock()
 
         mock_slack_notifier = SlackNotifier(None, None)
-        mock_slack_notifier.sendSlackNotification = MagicMock()
+        mock_slack_notifier.send_slack_notification = MagicMock()
 
         ingest.ingest(test_event, test_config)
 
         mock_writeToElasticsearch.assert_called_once()
 
     @patch.object(FormatterFactory, 'get_formatter')
-    @patch.object(SlackNotifier, 'sendSlackNotification')
-    def test_ingest_invalid_formatter(self, mock_sendSlackNotification, mock_get_formatter):
+    @patch.object(SlackNotifier, 'send_slack_notification')
+    def test_ingest_invalid_formatter(self, mock_send_slack_notification, mock_get_formatter):
```

Comment:
Would recommend to be consistent and revert all the `mock_send_slack_notification` to `mock_sendSlackNotification` as well.